### PR TITLE
[Doctests] Make TFRoberta-like meaningfull 

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Similar to #16363, but for TF

I made sure the doctests run without failure, but 2 models currently use `from_pt=True` --> need to convert and upload the TF ckpts.

@patrickvonplaten

### CODE REVIEW

File: src/transformers/utils/doc.py [hunk 1]
```diff
@@ -582,16 +591,29 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> import tensorflow as tf
 
     >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
+    >>> model = {model_class}.from_pretrained("{checkpoint}", from_pt=True)
```
Comment (ydshieh) [inline src/transformers/utils/doc.py:594]: TODO: upload TF checkpoint and remove from_pt

File: src/transformers/utils/doc.py [hunk 2]
```diff
@@ -560,17 +560,26 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
...
```
Comment (ydshieh) [inline src/transformers/utils/doc.py:563]: TODO: upload TF checkpoint and remove from_pt

File: src/transformers/utils/doc.py [hunk 3]
```diff
@@ -625,11 +655,24 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
...
     >>> inputs = tokenizer("The capital of France is {mask}.", return_tensors="tf")
-    >>> inputs["labels"] = tokenizer("The capital of France is Paris.", return_tensors="tf")["input_ids"]
+    >>> logits = model(**inputs).logits
 
-    >>> outputs = model(inputs)
-    >>> loss = outputs.loss
-    >>> logits = outputs.logits
+    >>> # retrieve index of {mask}
+    >>> mask_token_index = tf.where(inputs.input_ids == tokenizer.mask_token_id)[0][1]
+
```
Comment (ydshieh) [inline src/transformers/utils/doc.py:662]: not very clean here. There should be a better way.

File: src/transformers/utils/doc.py (hunk 2 above)
Comment (patrickvonplaten) [reply src/transformers/utils/doc.py:563]: yeah we can't do this here :-) Think if you upload the models under your name or you find other well-working TF ones that be perfect

File: src/transformers/utils/doc.py (hunk 3 above)
Comment (patrickvonplaten) [reply src/transformers/utils/doc.py:664]: think that's fine ;-)

**Comment by ydshieh** [comment]
@patrickvonplaten I uploaded the TF checkpoints and removed `from_pt`.

I will need to check and apply Sylvain's suggestion in your PR to finalize this PR.

**Review by patrickvonplaten — APPROVED** [review]
Perfect :-)

**Pull request merged**